circtpanamabinding/package.mill

```diff
@@ -0,0 +1,108 @@
+package build.circtpanamabinding
+
+import mill._
+import mill.scalalib._
+import mill.scalalib.scalafmt._
+
+import build._
+
+object `package` extends RootModule with CIRCTPanamaBinding
+
+// Java Codegen for all declared functions.
+// All of these functions are not private API which is subject to change.
+trait CIRCTPanamaBinding extends HasJextractGeneratedSources {
```

Comment:
`HasJextractGeneratedSources`, you can add a `panama.mill` file storing it, In the future, a contribution point is push our `Jextract` + `Panama` as plugin to official mill plugin.